site/docs/v1/tech/client-libraries/netcore.adoc

```diff
@@ -37,11 +37,104 @@ include::docs/v1/tech/client-libraries/_static-patch-note.adoc[]
+
+==== .NET CLI
+
+To create a new project from a template, navigate to your new project directory and run the following command.
+
+```bash
+dotnet new [template-name] [options]
+```
+
+Where `[template-name]` is the name of the template you want to use from one of the following:
+
+- `fusionauthblazorserver` creates a new Blazor Server application with FusionAuth authentication and authorization.
+- `fusionauthmvcwebapp` creates a new MVC application with FusionAuth authentication and authorization.
+- `fusionauthwebapi` creates a new Web API application with FusionAuth authentication and authorization.
+
+Use `[options]` to provide your FusionAuth URL and FusionAuth Application Client Id. The following options are available:
+
+- `--issuer` is the fully qualified URL to your FusionAuth server. The default value is `\http://localhost:9011`.
+- `--client-id` is the link:/docs/v1/tech/core-concepts/applications[Client Id] associated with your application. The default value is `3c219e58-ed0e-4b18-ad48-f4f92793ae32`.
+- `--port` is the port to run on under https://learn.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?view=aspnetcore-7.0[Kestrel], using HTTPS. The default value is `5001`. This can be changed after installation in the `appsettings.Development.json` file in the root directory of the project and `launchSettings.json` in the `Properties` directory of the project.
+
+==== Visual Studio for macOS
+
+FusionAuth .NET templates require Visual Studio for macOS version 17.6 or higher. 
+
+To create a new project from a template, first create or open a Solution. Then select [breadcrumb]#File -> New Project#. In the "New Project" dialog, select [breadcrumb]#Custom# from the left-hand menu. Select the FusionAuth template you want to use and click [uielement]#Continue#. Fill in the required information, including the fully qualified URL to your FusionAuth server and your FusionAuth Application Client Id. Click [uielement]#Continue#.
+
+Set the project name and click [uielement]#Create#.
+
+==== Visual Studio for Windows
+
+FusionAuth .NET templates require Visual Studio for Windows version 17.6 or higher. 
+
+To create a new project from a template, first create or open a Solution. Then select [breadcrumb]#File -> New Project#. In the "New Project" dialog, select "FusionAuth" from the "All project types" dropdown. Select the FusionAuth template you want to use. Set the project name and click [uielement]#Next#. Fill in the required information, including the fully qualified URL to your FusionAuth server and your FusionAuth Application Client Id.
+
+Click [uielement]#Create#.
+
+=== Testing
```

Comment:
@mark-robustelli 

> I couldn't seem to get the webAPI to work. I tried what I thought were valid API tokens but could not seem to get it to run. got 401 everytime. I maybe missing something, but think that needs to be more clear.

I have a dded a [script](https://github.com/FusionAuth/fusionauth-example-client-libraries/blob/4952a806e6bf0d50dbc970913155c13ead31fba1/dotnet/Kickstart/Program.cs) to help with testing the web API and added a section on testing to doc and to the README  https://github.com/FusionAuth/fusionauth-example-client-libraries/blob/5461e76a034b84761297f4bd4847103b37e9b6c9/dotnet/Kickstart/README.md#testing

Reply:
Ok, I ran the tests and it seems to work.